mod1/src/file-2.xoo

```diff
@@ -0,0 +1,7 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// violate HasTag rule: xoo on line 7
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:39485/organizations/org4/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:39485/project/issues?id=sample&pullRequest=6202&issues=AWjs-4ADXII8aNl01Mib&open=AWjs-4ADXII8aNl01Mib)

[ISSUE_KEY]: AWjs-4ADXII8aNl01Mib